Makefile

```diff
@@ -22,10 +22,10 @@ docker-web-server:
 	docker run -p 3000:3000 wps/web
 
 docker-shell-api:
-	# Shell into the dev container.
-	docker-compose run --rm api bash
+	# Shell into the running dev container.
+	docker compose exec api bash
```

Comment:
This is not a command on my machine but the original `docker-compose` is

Reply:
`docker-compose` is an implementation of Docker Compose V1, and has been deprecated. If you don't have `docker compose` it means you're running an older version of docker, and you should upgrade. `docker-compose` is an additional dependency, so apart from everything else, it simplifies the development environment.

I don't think we should continue to use a deprecated version, and should transition to v2.
https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command
https://www.docker.com/blog/announcing-compose-v2-general-availability/

Reply:
Huh weird. I'm running: `Docker version 20.10.16, build aa7e414fdc`

But I use the `limactl` wrapper which I guess just supports the old command.

Reply:
Maybe your `limactl` needs an update? lima with nerdctl seems to support it - so it's strange that it wouldn't work with docker?

Reply:
I needed to manually symlink the `docker compose` command as a plugin:

Compose is now a Docker plugin. For Docker to find this plugin, symlink it:    mkdir -p ~/.docker/cli-plugins    ln -sfn $(brew --prefix)/opt/docker-compose/bin/docker-compose ~/.docker/cli-plugins/docker-compose
--

https://formulae.brew.sh/formula/docker-compose